File: plugin/assets/lang/properties.c
set syntax  =c
set makeprg =LANGUAGE=en\ CFLAGS='-g\ -Wall'\ make\ %:r

"r'un
map <Leader>mr :!./%:r<CR>
"compile & run (make a'll)
map <Leader>ma :w<CR>:make && ./%:r<CR>

" vim: set ts=8 sw=4 tw=0 ft=vim : 

File: plugin/assets/lang/properties.java
set syntax  =java
set makeprg =javac\ %

"r'un
map <Leader>mr :!java %:r<CR>
"compile & run (a'll)
map <Leader>ma  :w<CR>:make && java %:r<CR>
"debug without arguments
map <Leader>md  :w<CR>:make && jdb %:r<CR>
"debug with arguments
map <Leader>mda :w<CR>:make && jdb %:r<SPACE>

let java_highlight_all       = 1
let java_highlight_functions = "style"
let java_allow_cpp_keywords  = 1

" vim: set ts=8 sw=4 tw=0 ft=vim :
